cli/maintainer/src/commands_multisig.rs

```diff
@@ -1711,3 +1724,24 @@ fn transfer_token(
         instruction,
     )
 }
+
+fn propose_upgrade_authority_change(
+    config: &mut SnapshotConfig,
+    opts: &ProposeUpgradeAuthorityChangeMultisigOpts,
+) -> Result<ProposeInstructionOutput> {
```

Comment:
We need to implement automatic test cases for this function, that verify  - we successfully with correct transaction proposal and we fail with wrong transaction proposal.